vision/tests/system.py

```diff
@@ -33,6 +33,10 @@
 LANDMARK_FILE = os.path.join(_SYS_TESTS_DIR, "data", "landmark.jpg")
 TEXT_FILE = os.path.join(_SYS_TESTS_DIR, "data", "text.jpg")
 FULL_TEXT_FILE = os.path.join(_SYS_TESTS_DIR, "data", "full-text.jpg")
+USER_PROJECT = os.environ.get("GOOGLE_CLOUD_TESTS_USER_PROJECT")
+# Valid locations include 'us-west1', 'us-east1', 'asia-east1', 'europe-west1'
+LOCATION = os.environ.get("GOOGLE_CLOUD_TESTS_LOCATION")
```

Comment:
Is it alright for tests to run against only one of these locations?

Reply:
Yes, running against 1 location is fine. I removed the environment variable and hard-coded "us-west1" as the location.